src/lib/storage/extensions.rb

```diff
@@ -20,26 +20,21 @@
 # find current contact information at www.suse.com.
 
 require "storage"
-require "expert-partitioner/icons"
+require "expert_partitioner/icons"
 
 include Yast::UIShortcuts
 
-
 module Storage
-
-
   def self.silence
-    silencer = ::Storage::Silencer.new()
+    silencer = ::Storage::Silencer.new
     yield
   ensure
     silencer.turn_off
   end
 
-
   class Device
-
+    # rubocop:disable Metrics/CyclomaticComplexity
```

Comment:
actually it make sense to say that this method too complex. Unless there is `Right` call to size, which looks a bit inconsistent to rest you can use constant with `FIELD_ID_TO_NAME = [ sid: N_("Storage ID") ]` and then here `fields.map { |f| _(FIELD_ID_TO_NAME[f]) }`. If right still have to be used, still for me it is easier to have here this hash defined as variable (having one value containing Right) and then just look at it. It is more effective and also easier to read and extend.